bin/crew

```diff
@@ -1067,6 +1067,29 @@ def compress_doc(dir)
   end
 end
 
+def determine_conflicts(dir, pkg)
+  conflicts = []
+  if File.file?("#{dir}/filelist")
+    if File.file?("#{CREW_META_PATH}#{pkg}.filelist")
+      puts 'Checking for conflicts with files from installed packages...'
+      conflictscmd = `grep --exclude #{CREW_META_PATH}#{pkg}.filelist --exclude #{CREW_META_PATH}*_build.filelist -Fxf #{dir}/filelist #{CREW_META_PATH}*.filelist`
+      conflicts = conflictscmd.gsub(/(\.filelist|#{CREW_META_PATH})/, '').split("\n")
```

Comment:
```ruby
conflicts = File.basename(conflictscmd, '.filelist').lines(chomp: true)
# conflicts.reject!(&:empty?) can be removed
```

Reply:
I don't think that accounts for BOTH gsub replacements though...

And I think the point of `conflicts.reject!(&:empty?)` is to then remove any lines which end up blank... though I agree that I can't imagine a situation where this would lead to a blank line... but better to be safe?

Reply:
Still need to remove CREW_META_PATH from the fields, because that will always conflict...
`gsub(/(\.filelist|#{CREW_META_PATH})/, '')`

Reply:
We need the path in each line though. Every file + path has to be unique... because the `def remove` code is checking to see if a pathname + file in the conflicts array matches a file about to be deleted.

Reply:
Never mind, it seems that I completely misunderstand the usage of this line (I thought it is used to handle the file paths but not the message outputted by `grep`) 😅